doc/source/release-notes/Percona-Server-8.0.27-18.rst

```diff
@@ -0,0 +1,70 @@
+.. _PS-8.0.27-18:
+
+================================================================================
+*Percona Server for MySQL* 8.0.27-18
+================================================================================
+
+:Date:  2022
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.27-18
+includes all the features and bug fixes available in the
+`MySQL 8.0.27 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+in addition to enterprise-grade features developed by Percona.
+
+*Percona Server for MySQL* is a free, fully compatible, enhanced and open
+source drop-in replacement for any *MySQL* database. It provides superior
+performance, scalability and instrumentation.
+
+*Percona Server for MySQL* is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads, and
+delivers greater value to *MySQL* server users with optimized performance,
+greater performance scalability and availability, enhanced backups and
+increased visibility. `Commercial support contracts are available
+<https://www.percona.com/services/support/mysql-support>`__.
+
+Release Highlights
+=================================================
+
+The following list are some of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated.  Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
```

Comment:
deprecated.  Support -> delete the space

Reply:
Done.